src/ui/SettingsButton.tsx

```diff
@@ -6,27 +6,184 @@ import {
   PopoverBody,
   ButtonGroup,
   Text,
+  Circle,
 } from '@chakra-ui/react';
-import { Navigator, ReaderState } from '../types';
+import {
+  Navigator,
+  PdfReaderState,
+  HtmlReaderState,
+  PDFActiveReader,
+  HTMLActiveReader,
+} from '../types';
 import { Icon, IconNames } from '@nypl/design-system-react-components';
 
 import Button from './Button';
 import ToggleButton from './ToggleButton';
 import ToggleGroup from './ToggleGroup';
 import useColorModeValue from './hooks/useColorModeValue';
 
-export default function SettingsCard({
+const PDFSettings = ({
+  navigator,
+  readerState,
+  paginationValue,
+}: {
+  navigator: Navigator;
+  readerState: PdfReaderState;
+  paginationValue: string;
+}): React.ReactElement => {
```

Comment:
Consider using object destructuring in the PDFSettings & HTMLSettings components with `navigator` and `readerState`. Ultimately I think this is a stylistic preference, so feel free to ignore this suggestion.